pom.xml

```
@@ -129,7 +129,8 @@
 				<module>erlang</module>
 				<module>evm-bytecode</module>
 				<module>fasta</module>
-                <module>fdo91</module>
+				<module>fbx</module>
+        <module>fdo91</module>
```

Please fix consistent whitespaces (tabs instead spaces for this case):

```suggestion
				<module>fbx</module>
				<module>fdo91</module>
```